src/sidebar/index.js

```diff
@@ -257,5 +244,8 @@ function startAngularApp(config) {
 fetchConfig(appConfig).then(config => {
   startAngularApp(config);
 }).catch(err => {
-  console.error(err);
+  // Report error. This will be the only notice that the user gets because the
+  // sidebar does not currently appear at all if the Angular app fails to
+  // start.
+  console.error('Failed to start Hypothesis client: ', err);
```

Comment:
I think we may want a user visible error (not just in the dev console) at some point. And I think we definitely want to log a crash report somewhere when this happens -- this should never be happening in production, and we need to know if it is happening, even if it's only happening a few times. Of course Sentry is currently disabled for the client. I think this can come in follow-on PRs, but I think it's a pretty desirable thing to have, if it's not working we want to know about it.

Reply:
Raven should send `console.error` reports AFAIK - but first we have to fix Sentry for the client as you noted.